boa_interner/src/interned_str.rs

```diff
@@ -0,0 +1,70 @@
+use std::{borrow::Borrow, ptr::NonNull};
+
+/// Wrapper for an interned str pointer, required to
+/// quickly check using a hash if a string is inside an [`Interner`].
```

Comment:
It seems that the link here doesn't work, as per the warning.

Reply:
Fixed